app/src/main/java/kr/ac/konkuk/gdsc/plantory/domain/entity/PlantDetail.kt

```diff
@@ -0,0 +1,10 @@
+package kr.ac.konkuk.gdsc.plantory.domain.entity
+
+data class PlantDetail (
+    val id: Long,
+    val dday: Int,
+    val nickname: String,
+    val species: String,
+    val description: String,
+    val createdAt: String
+)
```

Comment:
```suggestion
data class Plant(
    val id: Long,
    val dday: Int,
    val nickname: String,
    val species: String,
    val description: String,
    val createdAt: String
)
```
~Detail, ~Info 와 같은 접미사들은 가리키는 대상을 두고 중언부언하게 만들어요. 간략하게 표현하면 좋을 것 같습니다.

Reply:
저거 Plant라는 데이터 클래스가 이미 있어서 ,, 상세페이지용 데이터입니다 뭐가 좋으려나요